Source/Core/Core/PowerPC/BreakPoints.cpp

```diff
@@ -51,9 +52,16 @@ BreakPoints::TBreakPointsStr BreakPoints::GetStrings() const
     if (!bp.is_temporary)
     {
       std::ostringstream ss;
-      ss << std::hex << bp.address << " " << (bp.is_enabled ? "n" : "")
-         << (bp.log_on_hit ? "l" : "") << (bp.break_on_hit ? "b" : "");
-      bp_strings.push_back(ss.str());
+      ss << "$" << fmt::format("{:08x}", bp.address) << " ";
```

Comment:
```suggestion
      ss << fmt::format("${:08x} ", bp.address);
```

Condenses this to a single string insertion